spec/cc/engine/issue_spec.rb

```diff
@@ -36,6 +36,14 @@ module CC::Engine
           "This cop checks that the cyclomatic complexity of methods is not higher than the configured maximum."
         )
       end
+
+      it "sets a fingerprint for method/class offenses" do
```

Comment:
Worth asserting that it does *not* set a fingerprint for some other cop it shouldn't?